sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -75,7 +75,7 @@
 
 
 class TextAnalyticsClient(TextAnalyticsClientBase):
-    """The Text Analytics API is a suite of text analytics web services built with best-in-class
+    """The Language service API is a suite of natural language processing (NLP) skills built with the best-in-class
     Microsoft machine learning algorithms. The API can be used to analyze unstructured text for
     tasks such as sentiment analysis, key phrase extraction, entities recognition,
     and language detection, and more. No training data is needed to use this API - just bring your text data.
```

Comment:
with the custom tasks... is it true that no training data is needed?

Reply:
Going to remove those last few sentences, good point